Library/Source/NativeEngine.cpp

```diff
@@ -967,30 +970,22 @@ namespace Babylon
         bx::MemoryWriter writer(&mb);
         bimg::imageWritePng(&writer, image->m_width, image->m_height, image->m_size / image->m_height, image->m_data, image->m_format, false);
 
-        auto data = Napi::Int8Array::New(info.Env(), mb.getSize());
+        auto data = Napi::Uint8Array::New(info.Env(), mb.getSize());
         memcpy(data.Data(), static_cast<uint8_t*>(mb.more()), imageData->Image->m_size);
 
         return data;
     }
 
-    Napi::Value NativeEngine::LoadTexture(const Napi::CallbackInfo& info)
+    void NativeEngine::ImageToTexture(TextureData* const textureData, bimg::ImageContainer& image, bool invertY, bool mipMap) const
```

Comment:
nit: Functions typically start with a verb. Consider renaming to ConvertImageToTexture